pkg/cmd/root/command.go

```diff
@@ -131,3 +132,93 @@ func initConfig() {
 	targetProvider.TargetFile = filepath.Join(home, targetFilename)
 	factory.GardenHomeDirectory = home
 }
+
```

Comment:
I would propose to do the following:
* `pkg/cmd/root/command.go` -> `pkg/cmd/cmd.go`
* `pkg/cmd/root/command_whitebox_test.go` -> `pkg/cmd/cmd_whitebox_test.go `
* `pkg/cmd/root/root_suite_test.go` -> `pkg/cmd/cmd_suite_test.go`

Reply:
done